PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -849,4 +871,83 @@ export class ParaReportPublishService {
 
         return uuid.v5(violType + ruleId + msg + severity + lineHash + uri + order, namespace);
     }
+
+    private getReferenceSarifReports = async (): Promise<FileEntry[]> => {
+        const allBuildsForCurrentPipeline: Build[] = await this.buildClient.getBuildsForSpecificPipeline(this.projectName, this.definitionId);
+        let fileEntries: FileEntry[] = [];
+        if (!this.referenceBuild) {
+            tl.debug("No reference build has been set; using the last successful build with static analysis results");
```

Comment:
How about: No reference build has been set; using the last successful build as reference.